src/main/java/com/training/web/pages/flipkart/FlipkartHomePage.java

```diff
@@ -1,34 +1,31 @@
 package com.training.web.pages.flipkart;
 
+import com.codeborne.selenide.SelenideElement;
 import com.training.basepages.FlipkartBasePage;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.support.ui.ExpectedConditions;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$x;
 
 public class FlipkartHomePage extends FlipkartBasePage {
 
-    private By searchBox = By.xpath("//input[contains(@title,'Search for products')]");
-    private By popup = By.xpath("//button[@class='_2KpZ6l _2doB4z']");
-    private By submit = By.xpath("//button[@class='L0Z3Pu']");
+    private SelenideElement searchBox = $x("//input[contains(@title,'Search for products')]");
+    private SelenideElement popup = $x("//button[@class='_2KpZ6l _2doB4z']");
+    private SelenideElement submit = $x("//button[@class='L0Z3Pu']");
```

Comment:
Lets define this in a string rather than SelenideElement, and then use it in your methods